charts/gateway/values.yaml

```diff
@@ -0,0 +1,47 @@
+# Default values for gateway.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+traefik:
+  enabled: true
+  deployment:
+    providers:
+      kubernetesCRD:
+        enabled: true
+        ingressClass: "formance-gateway"
+      kubernetesIngress:
+        enabled: true
+        ingressClass: "formance-gateway"
+  service:
+    type: LoadBalancer
+#    annotations:
```

Comment:
is it really important to keep it for an oss helm chart ?

Reply:
I think it would be interesting to keep them somewhere, in our documentation?

Reply:
yes, in a "AWS configuration" section probably. I'll make so arrangement on the PR